core/src/tile/tile.cpp

```diff
@@ -73,7 +73,7 @@ void Tile::build(StyleContext& _ctx, const Scene& _scene, const TileData& _data,
                     auto* style = _scene.findStyle(rule.getStyleName());
 
                     if (style) {
-                        rule.eval(_ctx);
+                        if (!rule.eval(_ctx)) { continue; }
```

Comment:
Note for posterity: This *is* the desired future behavior (except for priority, which isn't actually required) and we'll move forward with this check in tangram-es